src/module/User/src/User/View/Helper/UserHelper.php

```diff
@@ -40,4 +41,39 @@ public function getAuthenticatedUserID() {
         $user = $this->userManager->getUserFromAuthenticator();
         return $user ? $user->getId() : '';
     }
+
+    public function getAuthenticatedUserName(){
+        $user = $this->userManager->getUserFromAuthenticator();
+        return $user ? $user->getUsername() : '';
+    }
+
+    public function isUserLoggedIn(){
+        return ($this->getAuthenticatedUserID() != '');
+    }
+
+    public function createJWSScrollback(){
+        $payload = array(
+            "iss" => "http://serlo.org",
```

Comment:
change this to HTTPS and correct subdomain if needed (see Instance\Entity\Instance and Ui/templates/instance/helper/select.phtml)